evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -1869,63 +1870,123 @@ def test_dates_needed_for_prediction_range(
 @pytest.mark.parametrize("set_coverage", [True, False])
 @pytest.mark.parametrize("add_decomposer", [True, False])
 @pytest.mark.parametrize("no_preds_pi_estimator", [True, False])
+@pytest.mark.parametrize("features", [True, False])
```

Comment:
We might need some inline comment coverage on these.  It always takes me a while to understand `no_preds_pi_estimator` and what that's testing for.  Now I'm a little confused what `features` being True or False means.

Reply:
👍 I've renamed `features` to `featuretools_first` and `no_preds_pi_estimator` to `ts_native_estimator` and added a couple comments, let me know what you think!